src/UI/Progress/index.tsx

```diff
@@ -9,7 +9,7 @@ import styles from './progressStyles.scss';
 
 const TOAST_PROGRESS_KEY = 'toastProgress';
 
-export interface ProgressProps extends WithClassnameType {
+export interface ProgressPropsType extends WithClassnameType {
```

Comment:
breaking change. better write TODO: rename in v3

Reply:
Reverted and noted. 👍